apps/api/package.json

```diff
@@ -1,14 +1,13 @@
 {
-  "name": "trading-simulator",
+  "name": "@recallnet/api",
   "version": "1.0.0",
   "description": "Solana Trading Simulator Server for hosting trading competitions",
   "main": "dist/src/index.js",
   "scripts": {
-    "lint": "eslint .",
+    "lint": "eslint . --max-warnings 0",
```

Comment:
Don't allow warnings. Eslint will now exit with code 1 if there are any warnings. All our packages are configured this way.